scripts/time_plot.py

```diff
@@ -1497,6 +1497,69 @@ def handle_websock_event(handlerkey,*args):
                 for thishandler in websockrequest_username.keys():
                     if (websockrequest_username[thishandler]==username):
                         send_websock_cmd('ApplyViewLayout('+'["'+'","'.join([fig['figtype'] for fig in html_viewsettings[username]])+'"]'+','+str(html_layoutsettings[username]['ncols'])+')',thishandler)
+        elif (args[0].startswith('holo')):#accepts 'holo', 'holo 3' where 3 is m003 as explicit reference antenna, else first track antenna is chosen as reference antenna
+            logger.info(repr(args))
+            antnumbers=[int(antnumberstr[1:]) for antnumberstr in telstate_antenna_mask]#determine all available inputs
+            if (len(antnumbers)==0):
+                send_websock_cmd('logconsole("No antenna inputs found or specified",true,true,true)',handlerkey)
+            else:
+                cbid=str(telstate['sdp_capture_block_id'])
```

Comment:
It's already a string - no need to `str`.

Reply:
Ok changed it to a comment. However I do not like this. Maybe you know offhand it is a string, but someone like me reading the code would not know if it is a string or a number and then wouldnt know how to handle it in the code.

Reply:
C is a much better language...